Added kubectl manifests to deploy webhook (#651)
jenkinsci/kubernetes-operator: website/content/en/docs/Getting Started/latest/installing-the-operator.md

* jenkins-home
* scripts
* init-configuration
* operator-credentials

## Validating Webhook 

> A validating webhook can be used in order to increase monitoring capabilities of the security issues. It will look for security vulnerabilities in the base and requested plugins. It can be easily installed via helm charts by setting webhook.enabled in values.yaml. 
> 
> **Note**: The webhook takes some time to get up and running. It's recommended to first deploy Operator and later Jenkins Custom Resource by using toggles in values.yaml
> If the installation happens with yamls, 
> First, install cert-manager:
> ```bash
> kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.1/cert-manager.yaml 
> ```
> It takes some time to get cert-manager up and running.
> Then, install the webhook and other required resources:
> ```bash
> kubectl apply -f https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/all-in-one-webhook.yaml
> ```
> Now, download the manifests for the operator and other resources from [here](https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/all-in-one-v1alpha2.yaml)and please provide these additional fields in the operator manifest:

Reviewer: ```suggestion
Validating webhook can be used in order to increase the Operator's capabilities to monitor security issues. It will look for security vulnerabilities in the base and requested plugins. It can be easily installed via Helm charts by setting webhook.enabled in values.yaml. 

**Note**: The webhook takes some time to get up and running. It's recommended to first deploy the Operator and later Jenkins Custom Resource by using toggles in `values.yaml`.

For the installation with yaml manifests (without using Helm chart), first, install cert-manager:
```bash
kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.1/cert-manager.yaml 
```
It takes some time to get cert-manager up and running.

Then, install the webhook and other required resources:
```bash
kubectl apply -f https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/all-in-one-webhook.yaml
```

Now, download the manifests for the Operator and other resources from [here](https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/all-in-one-v1alpha2.yaml) and provide these additional fields in the Operator manifest:
```

Reviewer 2: No, it's better to show the manifest here, for reference. But surround it with ```bash ```, as it's done in other places

Author: @SylwiaBrant ,Yeah but that way I won't be able to highlight specific parts, I am using html tags for formatting. It will still be rendered like a bash code block.
![yaml_webhook](https://user-images.githubusercontent.com/39304055/131504314-57024545-4830-4acb-a10d-43f813b68266.png)

Reviewer 2: hm. looks nice. Please fix the changes from the rest and we're ready to go :)

Reviewer 2: But maybe rather make them bold. It's loses readibility now.

Reviewer: @SylwiaBrant I didn't mean to suggest not showing the manifest and imo it's formatted well (though I agree with bolding the changes being a good idea), my suggestion ended with an empty listing because it kind of broke itself having a couple listings inside 😢
